File: Java/Car.java
package Java;

class Car {
    Integer id;
    String license;
    Account driver;
    private Integer passengers;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        if (passengers != null) {
            System.out.println("License: " + license + " - " + "Driver: " + driver.name  + " - " + "Passengers: " + passengers);
        }   
    }

    public Integer getPassengers() {
        return this.passengers;
    }

    public void setPassengers(Integer passengers) {
        if (passengers == 4) {
            this.passengers = passengers;
        } else {
            System.out.println("You need assign 4 passengers");
        }
    }
    
}

File: PHP/Car.php
<?php
require_once("Account.php");

class Car {
    public $id;
    public $license;
    public $driver;
    protected $passengers;

    public function __construct($license, $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function getPassengers() {
        return $this->passengers;
    }

    public function setPassengers($passengers) {
        if ($passengers == 4) {
            $this->passengers = $passengers;
        } else {
            echo "You need assign 4 passengers";
        }
    }

    public function printDataCar() {
        echo "
            License: $this->license
            Driver: {$this->driver->name}
            Passengers: $this->passengers
        
        ";
    }
    
}
?>
